ingestors/kafka/src/main/scala/hydra/kafka/programs/ConsumerGroupProgram.scala

```diff
@@ -0,0 +1,5 @@
+package hydra.kafka.programs
```

Comment:
Are you using this anywhere?

Reply:
An attempt at a refactor I strayed away from. Removing now.